webpack/configs/prod.ts

```diff
@@ -12,7 +12,7 @@ const ProdConfig: webpack.Configuration = {
 
     entry: {
         clarity: "./webpack/globalize.ts",
-        decode: "./decode/clarity.ts"
+        decode: "./webpack/globalize.ts",
```

Comment:
We do need two different files like you had before. We shouldn't have any reference to "decode" code in our primary clarity bundle that will get included for instrumentation. Decode is optional and should use a separate global file.